content/blog/events-documentation.md

```diff
@@ -0,0 +1,98 @@
+Title: New events documentation
+Date: 2022-10-04 13:00:00
+Author: Julien Alie
+Category: Wazo Platform
+Tags: Applications, Documentation
+Slug: new-events-documentation
+Status: published
+
+Hello Wazo community!  
+
+In line with launching our new [event documentation](https://wazo-platform.org/documentation) section on our website, in the last few months, 
+we have been busy refactoring our events subsystem in Wazo to make it simpler for developers and ourselves.  
+
+The first change, and most important, is that events are now fully _tenant-aware_, meaning they will never be dispatched to users outside
+of its intended tenant.
+
+We have also rewritten the way they work to streamline their use; events are now divided into 3 main types:
+
+#### Service-Level Events:
+Meant to be consumed by internal services (and plugins).  Will never be received by users.
+
+#### Tenant-Level Events:
+Meant to be consumed by all connected users of a tenant.
+These events usually represent changes at the tenant level on the stack (i.e: configuration setting changed)
+
+#### User-Level Events:
+Meant to be consumed by individual users.  
+These events usually represent specific actions targeting a user (i.e: joining a meeting or a conference, 
+receiving a chat message, receiving a call, etc)
+
```

Comment:
(SEO tips) Maybe we should convert to list. H4 in the middle of an article in note recommanded. We should follow title increment (h1, h2, h3, h4)

```suggestion
- **Service-Level Events:** Meant to be consumed by internal services (and plugins).  Will never be received by users.
- **Tenant-Level Events:** Meant to be consumed by all connected users of a tenant. These events usually represent changes at the tenant level on the stack (i.e: configuration setting changed)
- **User-Level Events:** Meant to be consumed by individual users. These events usually represent specific actions targeting a user (i.e: joining a meeting or a conference, receiving a chat message, receiving a call, etc)

```